tests/Console/Command/FixCommandTest.php

```diff
@@ -31,7 +31,7 @@ public function testEmptyRulesValue()
         $this->expectException(
             \PhpCsFixer\ConfigurationException\InvalidConfigurationException::class
         );
-        $this->expectExceptionMessageRegExp(
+        $this->expectExceptionMessageMatches(
```

Comment:
should be `static::`

Reply:
Why so? [It's an instance method.](https://github.com/sebastianbergmann/phpunit/blob/eafcb3a19339f99232354dbf949bd2f890f2491e/src/Framework/TestCase.php#L619-L622)

Reply:
Why this pointless nitpicking, mind you?

Reply:
The review process on this repo is rigorous. We are all grateful for contributions from everyone, however it is also in everyone's interest to make sure the code is of good quality, and works properly.

Reply:
I do not understand the request as well.
it was call on instance of object, it is still like that. the method is not static. why to change it?

as it's keeping the current status of repo, I'm happy to merge the PR without suggested change and have the change discussed separately (and maybe even merged, as separated PR?)

Reply:
Yeh, this review comment can be marked as done.